packages/venia-ui/lib/components/CartPage/ProductListing/quantity.js

```diff
@@ -29,41 +30,46 @@ export const QuantityFields = props => {
         maskInput
     } = talonProps;
 
+    const errorMessage = message ? <Message>{message}</Message> : null;
```

Comment:
This looks good - my only concern is that the message is a bit close to the field and not red.

Reply:
We stuck with the same UX from the dropdown treatment, which you recently implemented 😉  it looks pretty identical to me.

https://github.com/magento/pwa-studio/pull/2690#issuecomment-701398370